third-party/gasnet/Makefile

```diff
@@ -91,8 +91,9 @@ GASNET_CFLAGS += $(SHARED_LIB_CFLAGS)
 MPI_CC += $(SHARED_LIB_CFLAGS)
 endif
 
-# Disable hwloc until https://github.com/chapel-lang/chapel/issues/18050
-CHPL_GASNET_CFG_OPTIONS += --disable-hwloc
```

Comment:
I also think that if CHPL_HWLOC=none, this probably should still disable hwloc for GASNet?

Reply:
That wasn't one my original goals, but I agree that sounds like the right behavior if the user has explicitly disabled hwloc.